### PR TITLE
Create tooth patterns

### PR DESCRIPTION
**Pull request by megbalk**
Similar to the upper_tooth_pattern.yaml (see [PR](https://github.com/obophenotype/uberon/pull/2231)), this would create a pattern for the lower teeth

### CODE REVIEW

**Comment by megbalk** [comment]
pattern for naming teeth: 
“Secondary upper right first molar tooth”

**Comment by megbalk** [comment]
@nicolevasilevsky I'm not sure what the next steps are

File: src/patterns/dosdp-patterns/lower_tooth_pattern.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  # mandible UBERON:0001684
+  dentition: 'dentition_type'
+  # primary UBERON:0007116
+  # secondary UBERON:0007774
+  tooth: 'calcareous_tooth'
+  # incisor UBERON_0001098
+  # canine UBERON:0003674
+  # molar UBERON:0003655
+  # premolar UBERON:0007120
+  syn: "xsd:string"
```
Comment (rays22) [inline src/patterns/dosdp-patterns/lower_tooth_pattern.yaml:50]:
```suggestion
  your_syn: "xsd:string"
```
This is to fix QC error:

```
/usr/local/lib/python3.10/dist-packages/dosdp/validator.py:68: UserWarning:annotations.[0].vars has values ({'your_syn'}) not found in pattern variable list ({'dentition', 'syn', 'xrefs', 'jaw', 'tooth'}): 
```

File: src/patterns/dosdp-patterns/lower_tooth_pattern.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+name:
+  text: "%s lower %s"
+    # secondary lower first molar tooth
+  vars:
+    - dentition
+    - tooth
+
+def:
+  text: "A %s %s that is part of a lower jaw region."
+  # A secondary dentition molar tooth 1 that is part of a lower jaw region
```
Comment (rays22) [inline src/patterns/dosdp-patterns/lower_tooth_pattern.yaml:64]:
```suggestion
  # A secondary dentition molar tooth 1 that is part of a lower jaw region
  vars:
    - dentition
    - tooth
```
This is to fix QC error:

```
/usr/local/lib/python3.10/dist-packages/dosdp/validator.py:30: UserWarning:{'text': 'A %s %s that is part of a lower jaw region.', 'annotations': [{'annotationProperty': 'xref', 'value': 'xrefs'}]} is not valid under any of the given schemas
```

File: src/patterns/dosdp-patterns/upper_tooth_pattern.yaml [hunk 3]
```diff
@@ -18,53 +25,63 @@ pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/upper_teeth.yaml
...
+  tooth: 'calcareous_tooth'
   # incisor UBERON_0001098
   # canine UBERON:0003674
   # molar UBERON:0003655
   # premolar UBERON:0007120
-
-data_vars:
-  number: 'xsd:integer'
-  # 1-6
+  syn: "xsd:string"
```
Comment (rays22) [inline src/patterns/dosdp-patterns/upper_tooth_pattern.yaml:52]:
```suggestion
  your_syn: "xsd:string"
```
This is to fix an error.

**Review by rays22 — CHANGES_REQUESTED** [review]
* I have added suggestions of how to fix the QC errors.
* It is not obvious to me from the information in the PR what the anatomical justifications are for replacing `lower_teeth_pattern.yaml` with `lower_tooth_pattern.yaml`.

**Review by rays22 — APPROVED** [review]

**Pull request merged**